### PR TITLE
deletes driver_options, it was never used

### PR DESCRIPTION
**Pull request by jcarres-mdsol**
So it seems that the driver options were only used by the specs. Deleted them.
Also moved an instance method to a class method.

In general, less code and added some extra specs.

@mdsol/team-10

### CODE REVIEW

File: spec/lib/crawler_manager_spec.rb [hunk 1]
```diff
@@ -101,13 +118,43 @@
...
+    context 'There are no phantomjs processes running' do
+      let(:pids) { [] }
+      before do
+        allow_any_instance_of(Grell::CrawlerManager::PhantomJSManager).to receive(:running_phantomjs_pids).and_return(pids)
+      end
+      it 'no warning is logged' do
+        expect(Grell.logger).not_to receive(:warn)
+        described_class.cleanup_all_processes
+      end
+      it 'No process is killed' do
```
Comment (jfeltesse-mdsol) [inline spec/lib/crawler_manager_spec.rb:168]: as usual :trollface: , I think rubocop is not going to be 🆒  with this lack of new lines between stuff here.

Comment (jcarres-mdsol) [reply spec/lib/crawler_manager_spec.rb:168]: Nothing about that:
```
spec/lib/crawler_manager_spec.rb:127:121: C: Line is too long. [123/120]
        allow_any_instance_of(Grell::CrawlerManager::PhantomJSManager).to receive(:running_phantomjs_pids).and_return(pids)
                                                                                                                        ^^^
spec/lib/crawler_manager_spec.rb:147:121: C: Line is too long. [123/120]
        allow_any_instance_of(Grell::CrawlerManager::PhantomJSManager).to receive(:running_phantomjs_pids).and_return(pids)
                                                                                                                        ^^^
spec/lib/crawler_manager_spec.rb:158:1: C: Extra empty line detected at block body end.
```

Comment (jcarres-mdsol) [reply spec/lib/crawler_manager_spec.rb:168]: pushed better spacing (and solved the three problems above)

**Comment by jfeltesse-mdsol** [comment]
ルックス・グッド・トゥ・ミー

**Comment by ykitamura-mdsol** [comment]
LGTM 👍

**Pull request merged**